src/main/java/com/minecolonies/colony/jobs/JobBuilder.java

```diff
@@ -119,14 +120,11 @@ public WorkOrderBuild getWorkOrder()
     }
 
     /**
-     * Set a Work Order for this Job
+     * Set a Work Order for this Job.
      *
      * @param order Work Order to associate with this job, or null
      */
-    public void setWorkOrder(WorkOrderBuild order)
-    {
-        workOrderId = (order != null) ? order.getID() : 0;
-    }
+    public void setWorkOrder(WorkOrderBuild order) { workOrderId = (order == null) ? 0 : order.getID(); }
```

Comment:
Ehm, IntelliJ. I have no idea.

Btw, I can give multiple references where someone else used thr 1 line formatting.

I don't see any problem. Is this bullying?